### PR TITLE
Add stress tests for smart merge 

### PR DESCRIPTION
**Pull request by MikeStall**
Fix some real cloning bugs - fields weren't getting copied over for a clone.

### CODE REVIEW

File: src/PAModel/MsAppTest.cs [hunk 1]
```diff
@@ -14,6 +16,164 @@ namespace Microsoft.PowerPlatform.Formulas.Tools
...
+
+
+            // Save and verify checksums.
+            using (var temp1 = new TempFile())
+            using (var temp2 = new TempFile())
+            {
+                doc1.SaveToMsApp(temp1.FullPath);
+                doc2.SaveToMsApp(temp2.FullPath);
+
+                // bool same = Compare(doc1, doc2, Console.Out);
```
Comment (lesaltzm) [inline src/PAModel/MsAppTest.cs:101]: remove commented out code?

File: src/PAModel/MsAppTest.cs [hunk 2]
```diff
@@ -14,6 +16,164 @@ namespace Microsoft.PowerPlatform.Formulas.Tools
...
+                }
+
+                Directory.Delete(entropyDir, recursive: true);
+
+                (var doc2, var errors2) = CanvasDocument.LoadFromSources(temp1.Dir);
+                errors.ThrowOnErrors();
+
+
+#if false
+                // $$$ We get different checksums when deleting in-memory vs. files.
```
Comment (lesaltzm) [inline src/PAModel/MsAppTest.cs:147]: The directory also contains checksum.json and the appchecker file, in addition to what's covered by the in memory _entropy member.

File: src/PASopa/Program.cs [hunk 3]
```diff
@@ -34,6 +34,48 @@ static void Main(string[] args)
...
+                string msAppCommon = null;
+                foreach (var msAppPath in Directory.EnumerateFiles(msAppPathDir, "*.msapp", SearchOption.TopDirectoryOnly))
+                {
+                    // Merge test requires a 2nd app. Could do a full NxN matrix. But here, just pick the first item. 
+                    msAppCommon ??= msAppPath;
+
+                    Stopwatch sw = Stopwatch.StartNew();
+                    // bool ok = MsAppTest.DiffStressTest(msAppPath); // all pass!
+                    //bool ok = true;
+                    bool ok = MsAppTest.TestClone(msAppPath);
```
Comment (lesaltzm) [inline src/PASopa/Program.cs:60]: Is this supposed to be testing clone, or merge?

Comment (MikeStall) [reply src/PASopa/Program.cs:60]: I'll remove this code and just roll TestClone into the existin TestAll

**Review by lesaltzm — APPROVED** [review]
:ship:

**Pull request merged**